Solutions/Tenable Nessus/Data Connectors/Tenable_sc_NessusVM_API_FunctionApp.json

```diff
@@ -0,0 +1,129 @@
+{
+	"id":"NessusVMAPI",
+	"title":"Tenable.sc Vulnerability Management Reports",
+	"publisher":"Tenable",
+	"descriptionMarkdown":"The [Tenable.sc Nessus Vulnerability Management](https://www.tenable.com/products/tenable-sc) Report data connector provides the capability to ingest Scan reports events into Azure Sentinel through the REST API from the Tenable.sc vulnerability management solution (Managed on-premises). Refer to [API documentation](https://developer.tenable.com/reference) for more information. The connector provides ability to get events which helps to examine potential security risks, analyze your team's use of collaboration, diagnose configuration problems and more",
+	"additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on Kusto to work as expected. ​Follow the steps to use this Kusto functions alias **NessusVM** in queries and workbooks [Follow steps to get this Kusto functions>](https://aka.ms/)",
```

Comment:
Please replace the kusto function link https://aka.ms/ with https://aka.ms/NessusVM

Reply:
done